bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -81,15 +81,27 @@ async def apply_infraction(
         ctx: Context,
         infraction: _utils.Infraction,
         user: UserSnowflake,
-        action_coro: t.Optional[t.Awaitable] = None
-    ) -> None:
-        """Apply an infraction to the user, log the infraction, and optionally notify the user."""
+        action_coro: t.Optional[t.Awaitable] = None,
+        user_reason: t.Optional[str] = None,
+        additional_info: t.Optional[str] = "",
```

Comment:
I think it no longer makes sense for `additional_info` to be optional. The default is a string and we wouldn't want to get a `None` as its repr would show in the modlog.

Reply:
Addressed in acb6d73